recipes/QScreenCast/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "QScreenCast" %}
+{% set version = "0.1.31" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/QScreenCast-{{ version }}.tar.gz
+  sha256: 0bbaffb73905f3fb2a9f73f00ff2e6f32bc3ee5c663ef9a3d346ef5ee0153807
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [not win]
```

Comment:
Looking at the code I did not find anything that is Windows only but they do declare it as the only platform :-/

Reply:
Thanks for the quick review @ocefpaf! I think the package has some problems with the Qt version available in conda-forge, which does not have support for QMultimedia calls in Mac nor Linux.

Reply:
Makes sense. Maybe we should open an issue in the feedstock to track that.

Reply:
@ocefpaf, I made an issue for this :-)